Scripts/Extensions/Engine/AudioMotionExtensions.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using FriedSynapse.FlowEnt.Motions.Audio;
+
+namespace FriedSynapse.FlowEnt
+{
+    public static class AudioMotionExtensions
+    {
+        #region Pitch
+
+        /// <summary>
+        /// Lerps the pitch of an <see cref="AudioSource" /> from its current pitch to its current pitch plus the given <paramref name="value" />.
```

Comment:
use the same style of docs as in the value extensions

Reply:
👍